### DISCUSSION TITLE
Which firmware for Happymodel EPW5 RX?

### DISCUSSION THREAD

**Discussion by Miwer** [ExpressLRS/ExpressLRS#1631] [Q&A] [answered]
Hi,

I'm looking for information about which firmware is suitable for the Happymodel EPW5 receiver (5-channel PWM).
The configurator only lists the PP and EP RX targets, but no EPW5 target.

![ELRS-config](https://user-images.githubusercontent.com/145515/173666588-38086e38-4a63-49a6-a13f-68d6d48cb58a.png)

Is the EP target also valid for EPW5?
According to this link at happymodel.cn, I would guess that EP firmware is only compatible with EP1 and EP2, but I'm not sure if this is correct or not, and I cannot find other firmware specifically for the EPW5 receiver.

http://www.happymodel.cn/index.php/2021/04/10/elrs-rx-tx-firmware-for-happymodel-elrs-es24tx-pp_rx-ep1_rx-ep2_rx-stock-firmware-out-of-factory/

Maybe someone can help clarify this?

**Comment by pkendall64** [comment, marked as answer]
As it says on the happy model web site for that RX https://www.happymodel.cn/index.php/2021/12/29/happymodel-expresslrs-elrs-epw5-2-4ghz-5ch-pwm-rc-receiver-for-fixed-wing/

![Screen Shot 2022-06-15 at 6 57 43 AM](https://user-images.githubusercontent.com/512740/173667742-56595db5-2a51-4d29-bc21-2e9e8fb79bbc.png)

**Reply by jurgelenas** [reply]
Do you think we should add alias for this target?

**Comment by Miwer** [comment]
Oh well, I feel a bit silly now... 
It also says right on the update page (10.0.0.1) when the RX is in wifi mode... 

Thanks for replies, anyway ;)

![ELRS-epw5](https://user-images.githubusercontent.com/145515/173896478-af51a181-6ec8-4597-8709-30a0de3267f5.png)

**Comment by Davionics** [comment]
It's worth pointing out that the EPW5 Target has been added to the Happymodel 2.4 list in 3.3.0, which I discovered after some extensive faffing about...
![image_2025-03-10_000008347](https://github.com/user-attachments/assets/7375e441-9614-4251-9091-62aef3ccfa1b)
